bot.py

```diff
@@ -91,6 +91,29 @@ async def begin(ctx: commands.Context): # noqa: D103
 	await ctx.send("Profile created! `p!help` for more commands.")
 
 
+@bot.command(help='Add name to display specific Pokemon')
+async def show(ctx: commands.Context, single: Optional[str]): # noqa: D103
+	discord_id = ctx.author.id
+	base_msg = f"<@!{discord_id}> Here are all of your current Pokemon"
+	msg: Message = await ctx.send(base_msg)
+	#loads user class based upon their discord_id
+	user = await userprofile.load_profile(discord_id)
```

Comment:
The type checker is yelling at you for this. To fix it, add the following line below this one:
```python
assert isinstance(user, UserProfile)
```

Reply:
and you should remove that comment, because it states the obvious